code/datums/abilities/flock/flockmind.dm

```diff
@@ -375,7 +375,7 @@
 	if(..())
 		return TRUE
 	var/mob/living/intangible/flock/flockmind/F = holder.owner
```

Comment:
```suggestion
	var/mob/living/intangible/flock/flockmind/F = holder.get_controlling_mob()
```
without this, wouldn't F be null when `holder.owner` is a flockdrone?

Reply:
noo, the owner stays as the flockmind so should never be null unless the flockmind gets deleted